CONTRIBUTE.md

```diff
@@ -0,0 +1,143 @@
+
+```
+git add .
+git commit -m "Write message about your commit"
+```
+
+8. **Making sure you have the latest changes from the develop branch**
+
+It may so happen that since the last time you cloned the repo/took a pull from develop, some changes may be merged in the develop branch. So to be on the safer side, we should have those changes as well.
+
+In order to do that, we first checkout to **develop** branch by:
+
+```
+git checkout develop
+```
+
+Once we're in develop, it's time to take a pull:
+
+```
+git pull upstream develop
+```
+
+Now that our **local** develop branch is in sync with **remote** develop branch (of the Real Dev Squad Repository), we should let our branch know about the changes from the develop branch (if any). To do so we first checkout to our branch:
+
+```
+git checkout <branch-name>
+```
+
+Once we're in our branch, we **rebase** our branch on top of the current develop branch (we change the base of our branch, so that it appears as if we have worked from the time the latest changes were merged in the develop branch). To do so:
+
+```
+git rebase develop
+```
+
+You should solve the merge conflicts, if any. 
+
+
+9. **Pushing the code**
+
+Now that we have made our changes plus we have the latest changes made by other contributors, we should push our code from **local** branch to the same branch on our **GitHub fork**. We do so by:
+
+```
+git push origin <branch-name>
+```
+
+For example, if the branch name is `feat/login-form`, we enter `git push origin feat/login-form`
+
+The **origin** refers to your GitHub fork. You can check it by entering `git remote -v`, you should the link to your fork against **origin**.
+
+10. **Making a pull request**
+
+Your GitHub fork now has the changes, but you want those changes to be merged in the Real Dev Squad repository, right? There's a twist, you can't directly merge your code in the Real Dev Squad. Imagine you own a company whose code is open sourced, would you like if anyone could code make changes in the directly without asking you? For the same reason, **Pull Requests** exist. You `request` the repository maintainers/admins to `pull` your code in their repository.
+
+To make a pull request, go to your **forked repository** and you'll see **<branch-name> had recent pushes less than a minute ago** . Right next to it will be an option to **Compare & pull request**. Click on it, submit your pull request (also known as _PR_) explaining what you've done. Again, the PR title should be self-explanatory but concise. If you want to write details, you can add it in the description. If you're making some UI (User Interface) changes, please make sure to add a short screen recording. If that's not possible, at least add some screenshots.
+
+![how-to-create-pull-request](https://i.imgur.com/zYSuNY7.png)
+
+![before-making-pull-request](https://imgur.com/GTnYiKt.png)
```

Comment:
Good work here! I have one small suggestion, could you write the steps mentioned in the photo, instead of just pasting the photo in the file ? 
Also, the next time you make a PR, add labels which are mentioned in the corresponding issue and also add yourself to the assignee section. I have added the above two for now, but make sure you do that from the next time. 😃